DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/autorun/src/autorun.cpp

```diff
@@ -0,0 +1,176 @@
+#include <CL/sycl.hpp>
+#include <sycl/ext/intel/fpga_extensions.hpp>
+
+// dpc_common.hpp can be found in the dev-utilities include folder.
+// e.g., $ONEAPI_ROOT/dev-utilities//include/dpc_common.hpp
+#include "dpc_common.hpp"
+
+#include "autorun.hpp"
```

Comment:
Should we put this into the shared include directory?

Reply:
Yes, but it is not ready yet and I want to ensure this tutorial gets in.

Reply:
It's ready to be comitted for 2022.2 as soon as the 2022.1 label is created (early next week).  See https://github.com/oneapi-src/oneAPI-samples/pull/680.  It will be committed before this new sample is committed.

Some options for you:
1. Update this sample now to use the common include directory.
2. Wait until the common include directory PR is committed, then update this sample before we merge it.
3. We merge this sample as is, and sometime in 2022.2 you update this sample to move the header to the common include directory.

Your choice, just let me know which you plan to do.

Reply:
3 is my choice.